spark-jobs/src/main/scala/filodb/downsampler/chunk/BatchExporter.scala

```diff
@@ -211,9 +216,15 @@ case class BatchExporter(downsamplerSettings: DownsamplerSettings) {
    */
   private def getExportDatas(readablePartition: ReadablePartition,
                              partKeyMap: Map[String, String],
+                             rule: ExportRule,
                              userStartTime: Long,
                              userEndTime: Long): Iterator[ExportRowData] = {
-    val partKeyString = partKeyMap.map(pair => s"${pair._1}=${pair._2}").toSeq.sorted.mkString(",")
+    // Drop unwanted labels from the exported "labels" column.
+    val partKeyString = partKeyMap.filterKeys { label =>
```

Comment:
If multiple export rules are matched,  ``getRuleIfShouldExport`` returns the first match the drop label for the matched rule would be used.

Reply:
[see other comment]